src/util/tile-error-collection.js

```diff
@@ -0,0 +1,80 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var getValue = require('./get-object-value');
+var tileErrorImage = require('../../themes/img/tile-error.svg');
+
+var TILE_ERROR_IMAGE = 'data:image/svg+xml;base64,' + window.btoa(tileErrorImage);
+
+var TileErrorCollection = Backbone.Collection.extend({
+  initialize: function () {
+    this.running = false;
+    this.queue = new Backbone.Collection();
+
+    this.on('add', this._onAdd, this);
+  },
+
+  resetErrorTiles: function () {
+    this.running = false;
+    this.queue.reset();
+
+    var models = this.filter(function (model) {
+      this._deletedNode(model.get('tileDomNode'));
+    }.bind(this));
+
+    this.reset(models);
+  },
+
+  getError: function () {
```

Comment:
Is this something that has to be public? Are we going to expose that via the public API (eg: to BUILDER)?

Reply:
Yes, that was the idea, right now it has no use, but the idea was to use it in the next issues related to limits